jvm/featurespec-test/src/test/scala/org/scalatest/featurespec/AsyncFeatureSpecLikeSpec.scala

```diff
@@ -44,6 +44,10 @@ class AsyncFeatureSpecLikeSpec extends funspec.AnyFunSpec {
       class ExampleSpec extends AsyncFeatureSpecLike with ParallelTestExecution {
 
         //SCALATESTJS-ONLY implicit override def executionContext = org.scalatest.concurrent.TestExecutionContext.runNow
+        //SCALATESTNATIVE-ONLY implicit override def executionContext =
+        //SCALATESTNATIVE-ONLY   if (scala.scalanative.meta.LinktimeInfo.isMultithreadingEnabled)
+        //SCALATESTNATIVE-ONLY     scala.concurrent.ExecutionContext.Implicits.global
+        //SCALATESTNATIVE-ONLY   else super.executionContext
```

Comment:
Is it really needed? Scala Native does already use real, ForkJoinPool based ExecutionContext when multithreading is enabled or QueueExecutionContext in singlet-hreaded mode. Although I don't know what lives under `super.executionContext` 
https://github.com/scala-native/scala-native/blob/8ac5d27670389495ca3493213a61105df9a6f95b/scalalib/overrides-2.13/scala/concurrent/ExecutionContext.scala.patch#L16

Reply:
@WojciechMazur thanks fro you feedback. It uses https://github.com/scalatest/scalatest/blob/3.2.x-new/jvm/core/src/main/scala/org/scalatest/concurrent/SerialExecutionContext.scala under `super.executionContext`.

Reply:
Long story short: with this version it work with and without multithreading out the box.

I had encountered two issues https://github.com/scala-native/scala-native/issues/3846 and https://github.com/scala-native/scala-native/issues/3848 with enabled multithreading, but it seems to be Scala-Native related or/and my code (libuv integration) which I still "clean" for multithreading.